identity-wallet/src/state/reducers/history.rs

```diff
@@ -0,0 +1,32 @@
+use serde::{Deserialize, Serialize};
+use ts_rs::TS;
+
+#[derive(Clone, Serialize, Deserialize, Debug, PartialEq, Eq, TS)]
+#[ts(export)]
+pub enum EventType {
+    #[serde(rename = "connection_added")]
```

Comment:
Why rename? `"ConnectionAdded"` is perfectly idiomatic for the frontend as well.